.zappr.yaml

```diff
@@ -0,0 +1,9 @@
+approvals:
+  groups:
+    zalando:
+      minimum: 2
+      from:
+        orgs:
+          - "zalando"
+
```

Comment:
It's just any internal team name. For now, it has no importance or significances for the project..... if we were to use Zalando new CI-CD system it would be used to define what permissions we get (e.g. what pierone organisation we can push to).
As far as I know, there isn't a generic opensource team we can use.